src/scripts/twitter-handles.data.js

```diff
@@ -0,0 +1,32 @@
+const { en } = require('../lang/en')
+const { TwitterHandle } = require('../database/mongooseSchema')
+const { getTwitterHandles, saveTwitterHandles } = require('../TweetDbService')
+
+/* eslint-disable no-undef */
+const { TECH } = en.category
+const handles = [
+	{
+		name: 'Shrjan',
+		handle: '@itsmeshrijan',
+		category: TECH,
+		weight: 20
+	}
+]
+;(async function () {
+	try {
+		const initialHandles = await getTwitterHandles()
+		if (!initialHandles) {
+			const savedHandles = await saveTwitterHandles(handles)
+			console.log('saved handles', savedHandles)
+		} else {
+			console.log('Handles already exist data are ==', initialHandles)
+		}
+	} catch (error) {
+		if (error.code === 11000 || error.code === 11001) {
+			console.log('ignored duplicates')
+		} else {
+			console.log(error)
+		}
+	}
+	return null
+})()
```

Comment:
@shrijan-siris for some reason, the code is not waiting for all promises to be resolved. The process terminates immediately hence, the rows are not inserted.